resources/sass/layouts/_map.scss

```diff
@@ -1,25 +1,32 @@
 .ris-map {
     z-index: 1;
     display: flex;
-
-    @include breakpoint($bp-mobile-max) {
-        height: 30vh;
-        margin: auto;
-        width: 95%;
-    }
+    width: 100%;
 
     &-desktop {
         display: flex;
+        width: 100%;
 
         @include breakpoint($bp-mobile-max) {
-            display: none;
+            display: block;
+            // height: 100vh;
+            // display: none;
+        }
+    }
+    &__desktop {
```

Comment:
in this section it's unclear what mean `320px`, `175px`, `165px`, are there maybe vars for them or a way how to compute each of them?

Reply:
are added

Reply:
are added